--- ./drivers/input/touchscreen/gt9xx/gt9xx_old.c	2024-02-16 12:48:34.084117454 -0500
+++ ./drivers/input/touchscreen/gt9xx/gt9xx.c	2024-02-23 12:23:40.242444464 -0500
@@ -422,16 +422,13 @@
 *********************************************************/
 static void gtp_touch_down(struct goodix_ts_data* ts,s32 id,s32 x,s32 y,s32 w)
 {
-	if (gtp_change_x2y)
-		GTP_SWAP(x, y);
+	if (gtp_change_x2y)	GTP_SWAP(x, y);
 
-	if (!bgt911 && !bgt970) {
-		if (gtp_x_reverse)
-			x = ts->abs_x_max - x;
 
-		if (gtp_y_reverse)
-			y = ts->abs_y_max - y;
-	}
+	if (gtp_x_reverse)	x = ts->abs_x_max - x;
+
+	if (gtp_y_reverse)	y = ts->abs_y_max - y;
+	
 
 #if GTP_ICS_SLOT_REPORT
     input_mt_slot(ts->input_dev, id);
@@ -2664,8 +2661,8 @@
 	} else if (val == 911) {
 		m89or101 = FALSE;
 		bgt911 = TRUE;
-		gtp_change_x2y = TRUE;
-		gtp_x_reverse = FALSE;
+		gtp_change_x2y = FALSE;
+		gtp_x_reverse = TRUE;
 		gtp_y_reverse = TRUE;
 	} else if (val == 9110) {
 		m89or101 = FALSE;
